nss-day-cohort-48/fun-website-1-digital-drops
# Added headers and classes to our sections.

## Steps to Test
1. Pull this branch
2. Verify in VS code that the about us and contact us headers have classes.
3. Serve up in Chrome
4. Verify that the there are headers for the about us and contact us sections.
